beta/src/components/header/index.ts

```diff
@@ -433,14 +433,16 @@ class Header extends Component<HeaderProps> {
   }
 
   renderSearch (): HTMLElement {
+    const searchClasses: string = 'search flex-l flex-auto-l w-100-l justify-center-l'
     const search = {
       on: () => this.state.cache(Search, 'search').render({ tags: TAGS }),
       off: () => {
         const attrs = {
-          onclick: (e) => {
+          onclick: () => {
             this.local.machine.emit('search:toggle')
           },
-          class: 'js bn dn db-l bg-transparent'
+          class: `js bn dn db-l bg-transparent ${searchClasses}`,
+          style: 'height:3rem'
         }
         return html`
           <button ${attrs}>
```

Comment:
When you're already at it, the nested div inside this button shouldn't be needed (I tried removing it with success). The `flex items-center` classes can just be applied directly to the button instead. The icon and span can go straight inside the button.

Reply:
Nice, addressed here: https://github.com/resonatecoop/stream/pull/216/commits/9bc22dbb95fb1298f4dab7216258f3ace5ce59f5